packages/store/src/secure-store/cli-wrapper/win32-get-credential-script.ps1

```diff
@@ -0,0 +1,55 @@
+# Import the CredentialManager module
+Import-Module CredentialManager
```

Comment:
This module is out of date and no longer maintained.

Microsoft recommend to use this module instead:
https://www.powershellgallery.com/packages/Microsoft.PowerShell.SecretManagement/1.1.2